docs/deploy-hyperlane.mdx

```diff
@@ -181,6 +189,14 @@ The Kurtosis team is still building the secrets functionality of Kurtosis Cloud,
 
 ![Example Configuration Two](../static/img/kurtosis-example-config-two.png)
 
+:::warning
```

Comment:
I wonder if this is more confusing. IMO we should just clarify that this guide is specifically made to connect a chain that already has Hyperlane on it and a new chain

Reply:
good idea, will update to clarify this at the start of the guide